packages/flutter/test/painting/gradient_test.dart

```diff
@@ -445,7 +445,32 @@ void main() {
         Color(0x80777777),
         Color(0x80444444),
       ],
-    ),);
+    ));
+  });
+
+  test('LinearGradient withOpacity test - preserves transform', () {
```

Comment:
Most test descriptions don't include the word "test", since it's a bit redundant.

```suggestion
  test('LinearGradient withOpacity() preserves transform', () {
```